### PR TITLE
Fixed #29054 -- Fixed a regression where a queryset that annotates with geometry objects crashes.

### PR DESCRIPTION
**Pull request by felixxm**
https://code.djangoproject.com/ticket/29054

Regression in 19b2dfd1bfe7fd716dd3d8bfa5f972070d83b42f.

### CODE REVIEW

File: docs/releases/2.0.2.txt [hunk 1]
```diff
@@ -14,3 +14,6 @@ Bugfixes
 
 * Fixed incorrect foreign key nullification if a model has two foreign keys to
   the same model and a target model is deleted (:ticket:`29016`).
+
+* Fixed crash in ``QuerySet`` with multiple annotations when using geometry
```
Comment (timgraham) [inline docs/releases/2.0.2.txt]: Fixed a regression where a queryset that annotates with geometry objects crashes

**Comment by felixxm** [comment]
@timgraham Test failure on Oracle is not related with this PR. I'm trying to figure out how to create a regression test that we will pass on Oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
@timgraham Thanks for the review. I updated PR.

**Pull request merged**